html5ever/src/tokenizer/mod.rs

```diff
@@ -371,7 +371,11 @@ impl<Sink: TokenSink> Tokenizer<Sink> {
         TokenizerResult::Done
     }
 
+    #[inline]
     fn bad_char_error(&self) {
+        #[cfg(feature = "trace_tokenizer")]
+        trace!("  error");
```

Comment:
I think the usefulness of the `trace_tokenizer` feature is a bit questionable, but I didn't want to touch it for now. This matches what the `sh_trace!` macro would log before.